app/views/notes/show.html.erb

```diff
@@ -82,18 +84,18 @@
   <div class="d-print-none">
     <hr />
 
-    <% if @node.has_power_tag('prompt') && feature("prompt-#{@node.power_tag('prompt')}") != "" %>
+    <% if @node.has_power_tag('prompt', @preview_tags) && feature("prompt-#{@node.power_tag('prompt')}") != "" %>
       <% if current_user && (current_user.id == @node.uid || current_user.admin? || current_user.moderator? || current_user.is_coauthor?(@node)) %>
         <%= feature("prompt-#{@node.power_tag('prompt')}") %>
       <% end %>
     <% end %>
 
```

Comment:
@Sagarpreet should these lines be displayed? I am not really sure what `prompt` means..

Reply:
We have to mock `power_tag` model function also, but for now we can remove display of lines where  `power_tag` is used.
https://github.com/publiclab/plots2/blob/fc11ad2b1b03aab6e3f313e8eab1d701cae49714/app/models/node.rb#L398